examples/sgp4Prop/fsw/platform_inc/op_app_msgids.h

```diff
@@ -1,18 +1,14 @@
-
-
 #ifndef OP_APP_MSGIDS_H
 #define OP_APP_MSGIDS_H
 
-#define OP_FLAG_MID 		0x1800
-#define OP_CMD_MID  		0x1801                  
-#define OP_PERF_ID   		0x1802            
-#define OP_TICK_MID   		0x1803              
-#define OP_HK_MID     		0x1804     
-#define OP_PROPSTATE_MID 	0x1805
-
-#define ECI_TBL_MANAGE_MID 	0x1806
-#define ECI_SEND_HK_MID 	0x1807           
+#define OP_FLAG_MID         0x1AA0
```

Comment:
Tlm?

Reply:
Are you saying OP_FLAG_MID should be telemetry?

Reply:
For the SGP4 example, the [eci_interface.h](https://github.com/nasa/ECI/blob/cfe-6.6/examples/sgp4Prop/fsw/eci_inc/eci_interface.h#L23) specifies that OP_FLAG_MID is the value of ECI_FLAG_MID. ECI_FLAG_MID is used to [initialize ECI_AppData.FDCPacket](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/src/eci_app.c#L693), which is defined as [a App_FaultRep_SbMsg type](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/src/eci_app.c#L126) which is defined [here](https://github.com/nasa/ECI/blob/cfe-6.6/fsw/faultrep/app_faultrep.h#L139) as having a header `CFE_SB_TLM_HDR_SIZE` bytes long. So yea, looks like should be telemetry packet.

Reply:
Folded this into the new commits. I was more regimented about cmd/tlm designations.